### DISCUSSION TITLE
Docker and Manual Migrations from Changelog

### DISCUSSION THREAD

**Discussion by VictorioBerra** [pocketbase/pocketbase#4226] [Q&A] [answered]
What is the risk of using watchtower + an evergreen pocketbase container?

For example, spectado/pocketbase uses GitHub actions to pull the latest PocketBase release at midnight, and push that to dockerhub as tag "latest".

[watchtower](https://github.com/containrrr/watchtower) sees the new push, pulls the image, and recreates the container.

What are the changes PocketBase noted on their changelog that a manual migration is required and that the app fails to start?

**Comment by ganigeorgiev** [comment, marked as answer]
While I try to minimize the breaking changes, keep in mind that there is no backward compatibility guarantee before v1.0.0 and therefore it is recommended to always manually check the release notes before updating, especially if it is not a minor/bugfix release (eg. from v0.20.0 to v0.21.0).

The breaking changes can vary from one release to another and it could affect the database structure, Web APIs, the Go and JSVM exposed methods, etc.. The breaking changes are usually marked with "⚠️" in the [CHANGELOG.md](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md).
